sickbeard/scene_exceptions.py

```diff
@@ -273,9 +273,9 @@ def update_scene_exceptions(indexer_id, scene_exceptions, season=-1):
     # A change has been made to the scene exception list. Let's clear the cache, to make this visible
     if indexer_id in exceptionsCache:
         exceptionsCache[indexer_id] = {}
-        exceptionsCache[indexer_id][season] = scene_exceptions
+        exceptionsCache[indexer_id][season] = [se.decode('utf-8', 'ignore') for se in scene_exceptions]
```

Comment:
The question is: why is scene_exceptions a list of encoded items and not decoded items (like it should be)? It would be best to decode scene_exceptions at the "source". About decode('utf-8', 'ignore') I would personally prefer decode('utf-8', 'replace'), but - until we finish the transition to unicode everywhere - there is nothing wrong about it.